### ISSUE TITLE
GCM - Web compatibility

### ISSUE THREAD

**Issue by redDwarf03** [appsup-dart/crypto_keys#13]
Hello

First, thx for your lib !

I have an error with Uint64List in flutter Web (in pointycastle lib)
```
var length = Uint8List.view((Uint64List(2)..[0] = iv.length * 8).buffer);

""Error: Unsupported operation: Uint64List not supported on the web.
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/errors.dart 266:49  throw_
dart-sdk/lib/_internal/js_dev_runtime/patch/typed_data_patch.dart 115:5       new
packages/pointycastle/block/modes/gcm.dart 81:36                              [_computeInitialCounter]
packages/pointycastle/block/modes/gcm.dart 61:16                              prepare
packages/pointycastle/src/impl/base_aead_block_cipher.dart 217:5              reset
packages/pointycastle/block/modes/gcm.dart 47:11                              reset
packages/pointycastle/src/impl/base_aead_block_cipher.dart 117:5              init
packages/pointycastle/block/modes/gcm.dart 40:11                              init
packages/crypto_keys/src/symmetric_operator.dart 71:16                        encrypt
packages/archethic_lib_dart/src/utils/crypto.dart 673:16                      aesAuthEncrypt
packages/archethic_lib_dart/src/utils/crypto.dart 400:11                      ecEncrypt
packages/archethic_lib_dart/src/services/api_service.dart 483:46              newKeychainTransaction
packages/aewallet/util/keychain_util.dart 98:54                               createKeyChain
dart-sdk/lib/_internal/js_dev_runtime/patch/async_patch.dart 84:54            runBody
dart-sdk/lib/_internal/js_dev_runtime/patch/async_patch.dart 123:5            _async
packages/aewallet/util/keychain_util.dart 70:30                               createKeyChain
packages/aewallet/ui/views/intro/intro_backup_confirm.dart 547:28             createKeychain
dart-sdk/lib/_internal/js_dev_runtime/patch/async_patch.dart 45:50            <fn>
dart-sdk/lib/async/zone.dart 1653:54                                          runUnary
dart-sdk/lib/async/future_impl.dart 147:18                                    handleValue
dart-sdk/lib/async/future_impl.dart 766:44                                    handleValueCallback
dart-sdk/lib/async/future_impl.dart 795:13                                    _propagateToListeners
dart-sdk/lib/async/future_impl.dart 566:5                                     [_completeWithValue]
dart-sdk/lib/async/future_impl.dart 639:7                                     callback
dart-sdk/lib/async/schedule_microtask.dart 40:11                              _microtaskLoop
dart-sdk/lib/async/schedule_microtask.dart 49:5                               _startMicrotaskLoop
dart-sdk/lib/_internal/js_dev_runtime/patch/async_patch.dart 166:15           <fn>
"
```

I created an issue in pointycastle but this issue was closed directly without analyze :/
https://github.com/bcgit/pc-dart/issues/178/

Is it possible to change the call of symmetric_operator please ?

Thx

**Comment by rbellens** [comment]
I believe this is fixed in pointycastle: https://github.com/bcgit/pc-dart/pull/181

**Comment by redDwarf03** [comment]
Yes, it was fixed in the version 3.7.0 (cf https://github.com/bcgit/pc-dart/pull/181)
So, please, could you update crypto_keys with new dependencies in pub.dev please 🙏🏼
